src/components/controls/calendar/Calendar.js

```diff
@@ -1,40 +1,53 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+
+const styles = theme => ({
+  rootCalendar: {
+    [theme.breakpoints.down('sm')]: {
+      width: '5rem',
+    },
+    [theme.breakpoints.down('xs')]: {
```

Comment:
@azizmashkour, here you do not need `xs`, because `sm` down will include `xs` and the width is the same.